setup.py

```diff
@@ -37,6 +37,9 @@
         "src/clp_ffi_py/utils.cpp",
     ],
     extra_compile_args=[
+        "/std:c++20",
+        "/O2,"
```

Comment:
Why not `O3`?

Reply:
`O3` isn't [a valid option for MSVC](https://learn.microsoft.com/en-us/cpp/build/reference/o-options-optimize-code?view=msvc-170). 

MSVC's `/O2`, which "sets a combination of optimizations that optimizes code for maximum speed", is roughly the equivalent of GCC's `-O3`.

Reply:
kk, good to know that, thx!